### PR TITLE
[MIG] partner_phonecall_schedule: Migrated to v10

### PR DESCRIPTION
**Pull request by yajo**
@Tecnativa

### CODE REVIEW

File: partner_phonecall_schedule/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from __future__ import division
```
Comment (LoisRForgeFlow) [inline partner_phonecall_schedule/models/res_partner.py:5]: ~where is this used?~

edit: nevermind, I found an [explanation](https://stackoverflow.com/questions/7075082/what-is-future-in-python-used-for-and-how-when-to-use-it-and-how-it-works) :smile:

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Relevant commit with the minimal migration is a824adaa1006facc734be88a266b943efdb40557

@chienandalu @luismontalba please review

**Review by chienandalu — APPROVED** [review]
Tested on runbot :+1:

**Review by luismontalba — APPROVED** [review]
Tested on runbot

**Comment by yajo** [comment]
Merge?

**Comment by michotm** [comment]
Tested on runbot

**Comment by pedrobaeza** [comment]
You can use GitHub approval system (on the third tab)

**Review by michotm — APPROVED** [review]

**Comment by yajo** [comment]
I think this should be merged already.

**Pull request merged**